client/jetpack-cloud/sections/agency-dashboard/sites-overview/utils.ts

```diff
@@ -409,3 +409,15 @@ export const formatSites = ( sites: Array< Site > = [] ): Array< SiteData > | []
 		};
 	} );
 };
+
+/**
+ * Returns the product slug that can be purchased from the dashboard.
+ */
+export const getProductSlugFromProductType = ( type: string ): string | undefined => {
+	const slugs: Record< string, string > = {
+		backup: 'jetpack-backup-t2',
+		scan: 'jetpack-scan',
+	};
+
+	return slugs[ type ];
+};
```

Comment:
Out of curiosity. Do you know why we use hyphens rather than underscores for these slugs? IIRC, we use product slugs with underscores on the pricing page and other places.